index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 /* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(fn => {
```

Comment:
`conventional-changelog` still supports old node (I haven't had time to change it), so this needs to be es5 :(